pheps/phep-9999.md

```diff
@@ -0,0 +1,171 @@
+
+# Motivation
+<a name="motivation"></a>
+Currently, PyHC is at a crossroads for how to push forward as a community. There are two main schools of thought—originating from bi-annual meeting discussions, telecon chats, and further sidebar converstaion—with regards to what PyHC is and should be:  
+
+- a basic interpretation where PyHC is a collection, and listing, of open-soure Python packages with a relevance to Heliophysics and space physics, or
+- a standards-based interpretation where PyHC strives for compliance with our set standards, package interoperability, and standardization around one or more tools.   
+
+There is utility and validity to both approaches. A new PyHC package tiering system is intended to find a "best of both worlds" with the two ideas. Older, out-of-date, unmaintained, or publication-specific code could still have a place for listing and findability, while also allowing nuance between other packages that are more robust, trustworthy, maintained, and work toward the standards-based interpretation of being a PyHC package. 
+
+Further, this tiering system also allows users to get a clearer picture on what each PyHC package has to offer, and the state of the package's condition and development. Creation of a PyHC package tiering system also allows for justification for a myriad of benefits, for example, consideration for funding from a community travel fund, or extra help with improving a standards grouping grade.
+
+# Rationale
+<a name="rationale"></a>
+Decisions for tiering levels, requirements for each tier, and benefits accrued at each tier are based on conversations with the community (bi-annual meetings, telecons, etc.), and are listed here as a starting point for more discussion, likely to be refined in the future. Initially, ideas were presented to the community in a pyramid format. To make the differences between tiers more visible and understandable, it has been transformed into a spreadsheet format. 
+
+# PyHC Package Tiering Specifications
+<a name="specification"></a>
+There are four tiers proposed in this PHEP: Copper, Bronze, Silver, and Gold. These would replace the current structure of "Core Packages", "Other Packages", and "Un-evaluated Packages". See the table below for requirements associated with each tier:
+
+|       | **Gold** | **Silver** | **Bronze** | **Copper** | 
+| :-: | :----------: | :-----------: | :------------: | :-------------: |
+| Self-Evaluation Status | Completed | Completed | Completed | Not Done |
+| PyHC Standard Grades | Mostly green, some yellow | Several yellow, no red | A couple red | N/A |
+| pyOpenSci Review Status | Completed | In progress | Not Started | Not Started |
+| PHEP 3 Compliant? | Yes | Yes | Yes | No |
+| HSSI Metadata Compliant? | Yes | Mostly | Partially | Bare Minimum Met |
+| Software License | Fully compliant and excludes NOSA license | Fully compliant, allows NOSA license | Has non-recommended license (e.g., GPL) | Has no license | 
```

Comment:
I am not sure how I feel about relegating GPL packages to the third tier when they could be excellent in all other regards. I am aware of why we don't recommend copyleft licenses, but sometimes it can't be helped. GPL (etc) are very good licenses which people can have legitimate reasons for using. If we really had a package which that was it's only "bad" grade would we really want to relegate it to effectively the lowest tier when it's a real possibility the authors would be unable to fix it.

Reply:
I'm open to changing this. In your mind, what would the levels be @Cadair ?

Reply:
I might lump GPL and NOSA in the same category...agreed that making GPL "worse" than NOSA seems a bit much.

Reply:
I would be fine with Jon's idea on this. So, NOSA + copyleft licenses would be bronze tier? Silver and gold would both require a recommended license (not to include those)?

Reply:
Yeah, these suggestions work for me, I'll update.